symfony/routing/7.0/manifest.json

```
@@ -5,5 +5,10 @@
     "aliases": ["router"],
     "conflict": {
         "symfony/framework-bundle": "<7.0"
+    },
+    "env": {
+        "#1": "Configure how to generate URLs in non-HTTP contexts, such as CLI commands.",
+        "#2": "See https://symfony.com/doc/current/routing.html#generating-urls-in-commands",
+        "DEFAULT_URI": "http://localhost"
```

What about using the `SYMFONY_PROJECT_DEFAULT_ROUTE_URL` env var defined by the Symfony CLI, and fallback to `http://localhost`?
```suggestion
        "DEFAULT_URI": "${SYMFONY_PROJECT_DEFAULT_ROUTE_URL:-http://localhost}"
```

---

@Kocal, ah cool, I didn't know about this fallback syntax. I originally tried to use an Symfony env parser but these do not work in `.env`.

---

Created a PR: https://github.com/symfony-cli/symfony-cli/pull/642